sentry_sdk/integrations/flask.py

```diff
@@ -44,6 +44,7 @@
 except ImportError:
     raise DidNotEnable("blinker is not installed")
 
+FLASK_VERSION = None
```

Comment:
Is this line needed? We only seem to be using `FLASK_VERSION` inside the `setup_once` function?